views/components/PartnerApp.js

```diff
@@ -4,11 +4,13 @@ import SignUpForm from './SignUpForm';
 
 class PartnerApp extends React.Component {
   render() {
-    const { name, imageUrl, copy, partnerId } = this.props;
+    const { name, imageUrl, copy, partnerId, campaignKeyBeta, additionalFields } = this.props;
     const formDetails = {
       header: `${name}`,
       subhead: `${copy}`,
       partnerId: partnerId,
+      betaOptInPath: campaignKeyBeta,
+      extras: additionalFields,
```

Comment:
instead of doing this in ``` SignUpForm ```.js why not grab ``` campaignKey ``` here instead and pass it as props to ``` SignUpForm.js ```
if (!hideAlpha) {
 +    alphaView = <AlphaSignUpForm optin={PartnerService.getOptInPath(partnerId)} />;
 +  }
```